src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+// #region header
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+
+  &-logo {
+    display: block;
+  }
+}
+
+.nav {
+  &__list {
+    padding-inline-start: 0;
+    display: inline;
+  }
+
+  &__item {
+    display: inline-block;
+  }
+
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    margin-left: 17px;
+    height: 60px;
+    line-height: 60px;
+    display: block;
+  }
+}
+
+.is-active::after {
+  display: inline-block;
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--light-blue);
+  border-radius: 8px;
+}
+
+.is-active {
+  margin-left: 0;
+  height: 60px;
+  position: relative;
+  color: var(--light-blue);
+}
+
+.nav__link:hover,
+.nav__link:focus {
+  color: var(--light-blue);
+}
+
+// #endregion
+
+// #region main
+.main {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
```

Comment:
You are using a lot of magic numbers (200px, 408px, 46px, 48px, 50px, 40px). It's better to use variables for these values. This makes your code more maintainable, because you only need to change the value in one place if it needs to be updated.